src/OrchardCore.Modules/OrchardCore.Templates/Controllers/TemplateController.cs

```diff
@@ -81,18 +78,14 @@ public async Task<IActionResult> Index(PagerParameters pagerParameters, string d
 
             IEnumerable<KeyValuePair<string, Template>> templates;
             if (!string.IsNullOrEmpty(displayText))
-            {
-                 templates = templatesDocument.Templates.OrderBy(x => x.Key)
-                    .Where(template => template.Key.Contains(displayText, StringComparison.InvariantCultureIgnoreCase))
-                    .Skip(pager.GetStartIndex())
-                    .Take(pager.PageSize);
-            }
+                 templates = templatesDocument.Templates.Where(template => template.Key.Contains(displayText, StringComparison.InvariantCultureIgnoreCase));
```

Comment:
Orchard kódban mindig, de többsoros kifejezésnél a saját kódjainknál is kellenek a zárójelek. Amúgy pedig túl hosszú sor, tördeld.